services/vision/vision.go

```diff
@@ -351,11 +357,14 @@ func (vm *vizModel) ClassificationsFromCamera(
 	if err != nil {
 		return nil, errors.Wrapf(err, "could not find camera named %s", cameraName)
 	}
-	img, release, err := camera.ReadImage(ctx, cam)
+	imgBytes, mimeType, err := cam.Image(ctx, gostream.MIMETypeHint(ctx, utils.MimeTypeJPEG), extra)
```

Comment:
@bhaney wanted to pick your brain rq on if it's okay to default to JPEG like this

Reply:
You definitely want to test out on depth cameras, which never have jpeg available. That is the most important edge case

Reply:
If anything, why not just leave it blank?

Reply:
Also, you repeat the camera.Image call + rimage.Decode call a lot -- why not make a helper function in vision.go and use it everywhere instead?